h2o-docs/src/product/data-science/hglm.rst

```diff
@@ -0,0 +1,407 @@
+- :math:`e,u` are independent;
+- :math:`E \begin{bmatrix} u \\ e \\\end{bmatrix} = \begin{bmatrix} 0 \\ 0 \\\end{bmatrix} , cov \begin{bmatrix} u \\ e \\\end{bmatrix} = \begin{bmatrix} G & 0 \\ 0 & R \\\end{bmatrix} , G = \delta^2_u I_{(J*q)} , R = \delta^2_e I_{n \cdot} E \begin{bmatrix} u \\ e \\\end{bmatrix} \text{ is a size } (J * q + n) \text{ vector }, cov \begin{bmatrix} u \\ e \\\end{bmatrix} \text{ is a } (J * q + n) \times (J * q + n)` matrix. 
+
+In addition, we also consider the following alternate form:
+
+.. math::
+   
+   Y = X\beta + e^*, e^* = Zu + e \quad \text{ equation 4}
+
+where:
+
+.. math::
+   
+   cov(e^*) = V = ZGZ^T + R = \delta^2_u ZZ^T + \delta^2_e I_n \quad \text{ equation 5}
+
+We solve for :math:`\beta, u, \delta^2_u, \text{ and } \delta^2_e`.
+
+Defining an HGLM model
+----------------------
+Parameters are optional unless specified as *required*.
+
+Algorithm-specific parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **em_epsilon**: (Only available for EM method) Converge if beta/ubeta/tmat/tauEVar changes less (using L-infinity norm) than EM epsilon (defaults to ``0.001``).
+
+- **gen_syn_data**: If enabled, will add gaussian noise with variance specified in ``tau_e_var_init`` (defaults to ``False``).
+
+- **group_column**: The column that is categorical and used to generate the groups in HGLM (defaults to ``None``).
+
+- **initial_fixed_effects**: An array that contains the initial values of the fixed effects coefficient (defaults to ``None``).
+
+- **initial_random_effects**: An H2OFrame ID that contains the initial values of the random effects coefficient. The row names should be the random coefficient names (defaults to ``None``).
+	
+	.. note::
+
+		If you aren't sure what the random coefficient names are, then build the HGLM model with ``max_iterations=0`` and check out the model output field ``random_coefficient_names``. The number of rows of this frame should be the number of level 2 units. To figure this out, build the HGLM model with ``max_iterations=0`` and check out the model output field ``group_column_names``. The number of rows should equal the length of the ``group_column_names``.
+
+- **initial_t_matrix**: An H2OFrame ID that contains the initial values of the T matrix. It should be a positive symmetric matrix (defaults to ``None``).
+
+- **method**: Obtains the fixed and random coefficients as well as the various variances (defaults to ``"em"``).
+
+- `random_columns <algo-params/random_columns.html>`__: An array of random column indices to be used for ``HGLM``.
+
+-  `rand_family <algo-params/rand_family.html>`__: The Random Component Family specified as an array. You must include one family for each random component. Currently only ``rand_family=["gaussisan"]`` is supported.
+
+- **random_intercept**: If enabled, will allow random component to the GLM coefficients (defaults to ``True``).
```

Comment:
Please change 

- **random_intercept**: If enabled, will allow random component to the GLM coefficients (defaults to ``True``).

to

- **random_intercept**: If enabled, will generate a random intercept as part of the random effects coefficients.  (defaults to ``True``).